scripts/docker/gatkbase/Dockerfile

```diff
@@ -54,17 +58,20 @@ CMD ["bash"]
 ENV JAVA_LIBRARY_PATH /usr/lib/jni
 
 # Install miniconda
-ENV DOWNLOAD_DIR /downloads
-ENV CONDA_URL https://repo.anaconda.com/miniconda/Miniconda3-py310_23.1.0-1-Linux-x86_64.sh 
-ENV CONDA_MD5 = "32d73e1bc33fda089d7cd9ef4c1be542616bd8e437d1f77afeeaf7afdb019787"
-ENV CONDA_PATH /opt/miniconda
+# Following the instructions from https://docs.conda.io/projects/conda/en/latest/user-guide/install/rpm-debian.html
+# Note that this *does* install just miniconda, despite the package being named "conda"
+RUN curl https://repo.anaconda.com/pkgs/misc/gpgkeys/anaconda.asc | gpg --dearmor > /tmp/conda.gpg && \
+    install -o root -g root -m 644 /tmp/conda.gpg /usr/share/keyrings/conda-archive-keyring.gpg && \
+    gpg --keyring /usr/share/keyrings/conda-archive-keyring.gpg --no-default-keyring --fingerprint 34161F5BF5EB1D4BFBBB8F0A8AEB4F8B29D82806 && \
+    rm /tmp/conda.gpg && \
+    echo "deb [arch=amd64 signed-by=/usr/share/keyrings/conda-archive-keyring.gpg] https://repo.anaconda.com/pkgs/misc/debrepo/conda stable main" > /etc/apt/sources.list.d/conda.list && \
+    apt update && \
+    apt install conda && \
+    apt -y clean  && \
+    apt -y autoclean  && \
+    apt -y autoremove && \
+    . /opt/conda/etc/profile.d/conda.sh && \
+    conda -V
```

Comment:
Do you need to delete the conda trash that we were  removing before?
all the .a and .pyc files it makes when it builds stuff. (including itself)

Reply:
I've reverted all these changes